continuous_integration/environment-mindeps.yaml

```diff
@@ -0,0 +1,32 @@
+name: dask-distributed
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.8
+  - click=7.0
+  - cloudpickle=1.5.0
+  - cytoolz=0.10.1
+  - jinja2
```

Comment:
Now's probably a good time to choose a minimum version for `jinja2`, `pyyaml`, and `urllib3`

Reply:
Yup, was going to start off with whatever version first added support for 3.8 (assuming the packages aren't `noarch`) and work from there